test/DebuggerCommon/ES6_proto_invalidation.js.dbg.baseline

```diff
@@ -363,6 +363,12 @@
               "__lookupGetter__": "function <large string>",
               "__lookupSetter__": "function <large string>"
             },
+            "indexOf": {
+              "#__proto__": "function <large string>",
+              "prototype": "Object {...}",
+              "name": "string ",
+              "length": "number 1"
+            },

```

Comment:
The fact that the position of IndexOf in this list has changed is normal, as we don't load/declare IndexOf at the same place anymore. But it miss a few info compare to before. I actually fixed that earlier.

---
In reply to: [115389957](https://github.com/Microsoft/ChakraCore/pull/2898#discussion_r115389957) [](ancestors = 115389957)